src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-capabilities.mdx

```diff
@@ -18,14 +18,17 @@ To learn what capabilities a role has, go to the user management UI and view a s
 
 ## What are capabilities?  [#capabilities-roles]
 
-A New Relic full platform user with full permissions (for example, if they're in the default **Admin** group) is able to use all features of the platform. Some of the things you can do in New Relic we've made available as role **capabilities**: you can add them or remove them from a custom role, and we use them to differentiate between our default-available [standard roles](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#roles). The capabilities that we've made visible and available for selection are those we think organizations are likely to find useful to control.  
+A New Relic full platform user with all permissions (for example, as they would if they were in the default **Admin** group) is able to use all features of the platform. Some of the things you can do in New Relic we've made available as role **capabilities**: you can add them or remove them from a custom role, and we use them to differentiate between our default-available [standard roles](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#roles). The capabilities that we've made visible and available for selection are those we think organizations are likely to find useful to control.  
 
 There are a lot of New Relic functionalities that we **don't** make visible and available for selection. For example, there are various UI pages that you can access as any user, and that aren't gated by capabilities. For another example: a user in a group with **Organization** [administration settings](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#admin-settings) can configure organization-level settings and that power is not available for adding to a custom role. 
 
-Some other important points about role capabilities: 
+## Important points about capabilities [#important-points]
 
+Here are some other important points about role capabilities: 
+
+* **User type takes precedence.** User-type-restrictions override access given by a role's associated capabilities. For more on that, see [User access](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#user-type-groups-relation). 
```

Comment:
An example would be helpful here to add context to the sentence "User-type-restrictions override access given by a role's associated capabilities", and before sending them off to the doc link.